src/apps/Hosting/HostingFilesStore.js

```diff
@@ -66,6 +66,15 @@ export default Reflux.createStore({
     this.data.currentHostingId = hostingId;
   },
 
+  handleCloseOnUpload(event) {
+    event.preventDefault();
+    event.returnValue = 'Are you sure you want to close?';
```

Comment:
Probably not many people will see it, because we have support only for chrome and for chrome custom text isn't supported from v51.
![image](https://cloud.githubusercontent.com/assets/17015114/19142572/9e822b06-8b9e-11e6-89b4-c22a0f0d3205.png)

There will be default text from browser like:
![image](https://cloud.githubusercontent.com/assets/17015114/19142628/20467700-8b9f-11e6-8d8a-974da052fea0.png)

![image](https://cloud.githubusercontent.com/assets/17015114/19142635/2e3572bc-8b9f-11e6-91f7-19b35fe68340.png)